.gitignore

```diff
@@ -1 +1,2 @@
 build/
+.idea
```

Comment:
Ohh now I'm really curious, what is a .idea file?

Reply:
It's a directory my editor makes to store metadata. I can remove it from here and put it in my global gitignore if you want.